### PR TITLE
Change example code formatting

### PR DESCRIPTION
**Pull request by chvrl**
## Summary
Added spaces to example code to be consistent with rest of document, moved a colon out of a code block.

### CODE REVIEW

File: docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The following restrictions apply to implicitly-typed variable declarations:
 
 - `var` cannot be used on fields at class scope.
 
-- Variables declared by using `var` cannot be used in the initialization expression. In other words, this expression is legal`: int i = (i = 20);` but this expression produces a compile-time error: `var i = (i = 20);`
+- Variables declared by using `var` cannot be used in the initialization expression. In other words, this expression is legal: `int i = (i = 20);` but this expression produces a compile-time error: `var i = (i = 20);`
```
Comment (Youssef1313) [inline docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md]: I don't know if there is a difference between:

```csharp
int i = (i = 20);
int i = 20;
```

But I think it's worth to mention the difference if there is, or state that they're identical.

Comment (chvrl) [reply docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md]: Maybe someone can create an issue if it's confusing? I just wanted to help fix the markdown.

Comment (BillWagner) [reply docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md]: The extra assignment is need to have `i` on the right hand side of an assignment before it's type has been determined. That's why `var` isn't legal in `var = i = (i = 20);`. I'll approve this PR, and :shipit: now.

**Review by BillWagner — APPROVED** [review]
Thanks for making these updates @chvrl 

This does improve the quality of the markdown. I'll :shipit: now.

Thanks again @chvrl for making your first contribution to docs! 🎆

**Pull request merged**